src/compas/robots/model/joint.py

```diff
@@ -181,7 +181,6 @@ def get_urdf_element(self):
             'friction': self.friction,
         }
         attributes.update(self.attr)
-        attributes = dict(filter(lambda x: x[1], attributes.items()))
```

Comment:
This line would result in many joints have attribute-less dynamics tags `<dynamics />` which is not allowed, at least by pybullet.  So now they look like `<dynamics  friction="0.0" damping="0.0" />`.  Much easier than trying to eliminate the tag completely if it doesn't have non-default attributes.

Reply:
I was wondering what this line was doing :P